cmd/kubectl-gadget/utils.go

```diff
@@ -21,20 +21,44 @@ import (
 	"fmt"
 	"io"
 	"math/rand"
+	"os"
+	"os/signal"
+	"strings"
+	"syscall"
+	"time"
 
+	log "github.com/sirupsen/logrus"
 	"github.com/spf13/viper"
 
 	corev1 "k8s.io/api/core/v1"
 	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/fields"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/kubernetes/scheme"
 	restclient "k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
 	"k8s.io/client-go/tools/remotecommand"
 
+	"github.com/kinvolk/inspektor-gadget/cmd/kubectl-gadget/utils"
+	gadgetv1alpha1 "github.com/kinvolk/inspektor-gadget/pkg/api/v1alpha1"
 	"github.com/kinvolk/inspektor-gadget/pkg/factory"
+	"github.com/kinvolk/inspektor-gadget/pkg/k8sutil"
 )
 
+const (
+	GADGET_OPERATION = "gadget.kinvolk.io/operation"
+)
+
+func init() {
+	gadgetv1alpha1.AddToScheme(scheme.Scheme)
+}
+
+func init() {
+	rand.Seed(time.Now().UnixNano())
+}
+
```

Comment:
Can I ask what is the purpose/advantage in this specific case of separating this into two `init` functions rather than using a single one? 😅

Reply:
Not much: it is just that the 2 statements are unrelated to each other. Moving into a single init() function...